config/webpack.dev.config.js

```diff
@@ -102,6 +85,10 @@ module.exports = Merge.smart(commonConfig, {
   devServer: {
     host: '0.0.0.0',
     port: 18011,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
```

Comment:
This allows Studio devstack, which is running on another "domain": `localhost:18010`, to access the webpack-dev-server on this docker container (`dahlia.studio-frontend`). Theoretically this would allow any other website on the internet to access your SFE assets, but :man_shrugging: don't put anything secret there.